src/quotamodel.c

```diff
@@ -795,9 +795,9 @@ refresh_disk_quota_model(bool is_init)
 static void
 refresh_disk_quota_usage(bool is_init)
 {
-	bool  pushed_active_snap          = false;
-	bool  ret                         = true;
-	HTAB *local_active_table_stat_map = NULL;
+	volatile bool pushed_active_snap           = false;
+	volatile bool ret                          = true;
+	HTAB *volatile local_active_table_stat_map = NULL;
```

Comment:
It makes no sense to make this variable volatile, it is easier to move its declaration inside the PG_TRY block, because it is used only there.